drivers/gles3/shaders/screen_space_reflection.glsl

```diff
@@ -159,8 +159,8 @@ void main() {
 
 		if (depth > z_to) {
 			// if depth was surpassed
-			if (depth <= max(z_to, z_from) + depth_tolerance) {
-				// check the depth tolerance
+			if (depth <= max(z_to, z_from) + depth_tolerance && -depth < camera_z_far) {
```

Comment:
Consider use of extra brackets for readability.

Reply:
You mean so it doesn't look like depth_tolerance and -depth are attached? Just pushed something like that.

Reply:
This is all personal preference, but I find long expressions with no brackets difficult to parse. Computers are good at working out operator precedence, humans, not so much. :+1:

Reply:
Makes sense to me; I could see how the above could throw someone off.